### ISSUE TITLE
skipFirstRows not working?

### ISSUE THREAD

**Issue by kansuke231** [crealytics/spark-excel#90]
Hi all.

We use spark-excel for reading Excel files, and one thing which bothers us is that the optional property `skipFirstRows` does not seem to be working, that is, we would get a DataSet with full rows. The weird thing is that other optional properties e.g. `maxRowsInMemory` are completely fine. When I feed a string for the property like `option("maxRowsInMemory", "ThisIsString")`, it would yield an error. However, it's not the case for `skipFirstRows`, meaning it does not yield any error. 

Any thoughts on this? Thanks!

**Comment by nightscape** [comment]
Hi @kansuke231,
I'm refactoring that entire part to support writing to and reading from tables as well.
A beta version of that should be out this week.

**Comment by kansuke231** [comment]
Thanks @nightscape

**Comment by nightscape** [comment]
Hi @kansuke231,

I'm just publishing `0.11.0-beta1`. Once it has landed on Maven Central you can give it a try using the new [`dataAddress`](https://github.com/crealytics/spark-excel/tree/append_to_existing_file#data-addresses) option.

**Comment by kansuke231** [comment]
Hi @nightscape,

Thanks for uploading the beta version. However, I am seeing another error when I run `mvn  -DartifactVersion=1.0.251 clean compile test package` with the updated pom that includes
```
<dependency>
    <groupId>com.crealytics</groupId>
    <artifactId>spark-excel_2.11</artifactId>
    <version>0.11.0-beta1</version>
```

The error message looks like:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-shade-plugin:2.3:shade (default) on project [project-name]: Error creating shaded jar: null: IllegalArgumentException -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-shade-plugin:2.3:shade (default) on project [project-name]: Error creating shaded jar: null
```

The maven command above works for spark-excel version ` 0.9.18`, but versions above do not.
Any thoughts would be highly appreciated.

Thanks.

**Comment by nightscape** [comment]
If it doesn't work on 0.10.0 then it might have something to do with the upgrade of POI. Maybe your shading configuration refers to an artifact or package that has changed?

**Comment by nightscape** [comment]
I just released `0.11.0`. Please have a look at the corresponding `CHANGELOG` and the `README` about the new `dataAddress` option which supersedes `sheetName`, `startColumn`, `endColumn` and `skipFirstRows`.
